### PR TITLE
Adds --headers option

### PR DESCRIPTION
**Pull request by sanpii**
This also replaces the `-h` option, like httpie.

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -77,6 +77,10 @@ pub struct Cli {
     #[structopt(short = "S", long)]
     pub stream: bool,
 
+    /// Always stream the response body
```
Comment (ducaale) [inline src/cli.rs:80]: Can you update this comment?

**Review by ducaale — CHANGES_REQUESTED** [review]
Looks good. I've one comment

File: src/cli.rs (hunk 1 above)
Comment (blyxxyz) [inline src/cli.rs:80]: You forgot to change this one :slightly_smiling_face:

Comment (sanpii) [reply src/cli.rs]: Oops, done!

**Review by ducaale — APPROVED** [review]

**Comment by ducaale** [comment]
Thanks, @sanpii!

**Pull request merged**